### DISCUSSION TITLE
Question about value of the 'Ndir 'column

### DISCUSSION THREAD

**Discussion by moonquakes66** [PaddlePaddle/PaddleSpatial#62] [BaiduKDDCup2022] [answered]
The introduction says the meaning of Ndir is "nacelle direction, i.e., the yaw angle of the nacelle".But I find some values of Ndir is greater than 360°,so can I turn these values which is greater than 360° into these values substract 360?If can not,which is these values mean?

**Comment by zhoujingbo** [comment, marked as answer]
Hi，

Please refer to our dataset report. The range of nacelle direction is [-720, 720]. Thanks for your support. 

Regards
